addons/redis/scripts/redis-start.sh

```diff
@@ -140,6 +130,93 @@ init_or_get_primary_node() {
   done
 }
 
+init_or_get_primary_from_redis_sentinel() {
+  # check redis sentinel component env
+  if [ -z "$SENTINEL_COMPONENT_NAME" ]; then
+    # return default primary node if redis sentinel component name is not set
+    echo "SENTINEL_COMPONENT_NAME env is not set, try to use default primary node."
+    get_default_initialize_primary_node
+    return
+  fi
+
+  # parse redis sentinel pod list from $SENTINEL_POD_NAME_LIST env
+  if [ -z "$SENTINEL_POD_NAME_LIST" ]; then
+    echo "Error: Required environment variable SENTINEL_POD_NAME_LIST: $SENTINEL_POD_NAME_LIST is not set."
+    exit 1
+  fi
+
+  # get redis sentinel headless service name from $SENTINEL_HEADLESS_SERVICE_NAME env
+  if [ -z "$SENTINEL_HEADLESS_SERVICE_NAME" ]; then
+    echo "Error: Required environment variable SENTINEL_HEADLESS_SERVICE_NAME: $SENTINEL_HEADLESS_SERVICE_NAME is not set."
+    exit 1
+  fi
+
+  old_ifs="$IFS"
+  IFS=','
+  set -f
+  read -ra sentinel_pod_list <<< "${SENTINEL_POD_NAME_LIST}"
+  set +f
+  IFS="$old_ifs"
+  declare -A master_count_map
+  local default_redis_primary_host=""
```

Comment:
default_redis_primary_host -> first_redis_primary_host
default_redis_primary_port -> first_redis_primary_port

would be better

Reply:
> default_redis_primary_host -> first_redis_primary_host default_redis_primary_port -> first_redis_primary_port
> 
> would be better

fixed